plugins/CorePluginsAdmin/Commands/ActivatePlugin.php

```diff
@@ -37,6 +37,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return;
         }
 
+        if ($dependencies = $pluginManager->getPluginMissingDependenciesAsString($plugin)) {
```

Comment:
I wonder if we should directly use the PluginManager here to get the loaded plugin, and directly use dependency class. If we otherwise move everything that loads a plugin or something into Plugin\Manager we end up having hundreds of methods there. Also it adds a formatting method is in `Dependency` so that class is actually doing many things. If I may suggest an improvement I would say to rather add a method `getMissingDependencies` to the `Plugin` class and then do `Plugin\Manager->loadPlugin(pluginName)->getMissingDependencies()`.

Ideally we would have a class `MissingDependencies` that extends ArrayObject and is therefore iterable and also cares about the formatting. I'm saying this because we will soon move this part into composer but we can also do this change later and refactor it again soon.